src/cli/domain/package-components.js

```diff
@@ -12,12 +12,10 @@ var getUnixUtcTimestamp = require('../../utils/get-unix-utc-timestamp');
 var validator = require('../../registry/domain/validators');
 
 module.exports = function(){
-  return function(componentPath, minify, callback){
```

Comment:
I was wondering if we should have a minify option at all, but probably I'm missing some context. Why would we want to set minify, what are we trying to solve?